data/src/main/java/com/teamagam/gimelgimel/data/message/repository/MessageSenderMapper.java

```diff
@@ -0,0 +1,26 @@
+package com.teamagam.gimelgimel.data.message.repository;
+
+import com.teamagam.gimelgimel.data.message.repository.cloud.CloudMessagesSource;
+import com.teamagam.gimelgimel.data.response.adapters.ServerDataMapper;
+import com.teamagam.gimelgimel.domain.messages.MessageSender;
+import com.teamagam.gimelgimel.domain.messages.entity.ChatMessage;
+import io.reactivex.Observable;
+import javax.inject.Inject;
+
+public class MessageSenderMapper implements MessageSender {
```

Comment:
Why is it called MessageSenderMapper ?
What does it map ?
It seems like it maps nothing and only sends.
You can either call it MessageSenderData by convention or give it some other meaningful name